MANIFEST.in

```diff
@@ -8,6 +8,7 @@ include .pre-commit-config.yaml
 recursive-include docs *
 recursive-include tests *.py
 recursive-include tests hello-world-*
+recursive-include packaging *.py
```

Comment:
Well, when we get to this, we should drop this and change the setup.py to use find_packages. :)

Reply:
Or should we choose to something that uses `pyproject.toml` like flit?

Reply:
Works. Flit?

Off topic: I'd really like to make pip's performance better w/ PEP 517 packages, and adding them to pip's own vendoring stack makes things slower.

But that's more a pip enhancement that needs to be made than a packaging issue. :)

Reply:
Flit WFM.

Reply:
#340 for that.